gcp/plugin.go

```diff
@@ -36,6 +37,40 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 		ConnectionConfigSchema: &plugin.ConnectionConfigSchema{
 			NewInstance: ConfigInstance,
 		},
+		RateLimiters: []*rate_limiter.Definition{
+			// FIXME: API Requests per 100 seconds: 5,000
+			// https://cloud.google.com/memorystore/docs/redis/quotas#per-second_api_requests_quota
+			{
```

Comment:
@pdecat did you run into specific rate limit issues while testing out this table?

What would happen if we use the table without these rate limits?

Reply:
I actually only faced errors with the cluster API which has very low quotas. The retry code wouldn't suffice to handle those.

Also, removing the usage of matrix item function did help improve the situation as way less API calls are made.

Reply:
@pdecat do we still need the `FIXME` comments? As per your test results, if you feel that they are the optimum values, I think we can have them in.

Reply:
Given the answers I got about this in Slack, especially the fact that [most plugins will back off and retry](most plugins will back off and retry), I do believe those are reasonable values.

I've removed the FIXMEs.